Weather/Views/TimedForecastCellView.swift

```diff
@@ -12,6 +12,32 @@ class TimedForecastCellView: UICollectionViewCell {
     let timeLabel = UILabel()
     let temperatureLabel = UILabel()
     
+    var presenter: TimedForecastPresenter?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        presenter = TimedForecastPresenterImplementation()
```

Comment:
I don't think that cell view should have the access to the presenter. For view configuration there is `dataSource`.